src/ipc/ipc4/dai.c

```diff
@@ -308,6 +308,8 @@ void dai_dma_position_update(struct comp_dev *dev)
 	status.ipc_posn_data = llp_data;
 	dma_status(dd->chan, &status, dev->direction);
 
+	platform_dai_wallclock(dev, &dd->wallclock);
+
```

Comment:
@RanderWang @mmaka1 can you confirm when this dma_position_update happens? it's my understanding that it must happen on a 1ms basis, not just when there a DMA event.

Reply:
@plbossart yes, it is called by each dai_copy.